### PR TITLE
docs: add a link to our Github repository in the navbar

### PR DESCRIPTION
**Pull request by JohannesDienst-askui**
* Also, modify `versions.json` to only show versions above 0.2.6 in the versions dropdown in navbar 😉 
* Remove Why-Askui page
* Redirect directly to Getting Started Page from Docs-Homepage

### CODE REVIEW

File: docs/docs/general/02-Getting Started/write-your-first-instruction.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 sidebar_position: 3
 ---
 
-# Write Your First Automation
+# Write Your First Instruction
```
Comment (adi-wan-askui) [inline docs/docs/general/02-Getting Started/write-your-first-instruction.md:5]: Why do you think this is better than `Automation`?

I saw that we still talk about `automation` on this page. So if we change it here, I believe we should make it consistent across the page.

File: docs/docs/general/01-Introduction/why-askui.md [hunk 2]
```diff
@@ -1,21 +0,0 @@
----
```
Comment (adi-wan-askui) [inline docs/docs/general/01-Introduction/why-askui.md:1]: Was the removal of this file intentional? Can you shortly explain the reasoning to me?

File: docs/src/css/custom.scss [hunk 3]
```diff
@@ -29,6 +29,19 @@
...
+}
+
+.header-github-link:before {
+  content: '';
+  width: 24px;
+  height: 24px;
+  display: flex;
+  background: url("data:image/svg+xml,%3Csvg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12'/%3E%3C/svg%3E")
+  no-repeat;
+}
```
Comment (adi-wan-askui) [inline docs/src/css/custom.scss:43]: I would put the svg in its own file. Also, have you checked if the css of the theme icon button is reusable so they both use the same styling?

**Review by adi-wan-askui — CHANGES_REQUESTED** [review]
Nice and short :) Like it! Just have some questions so that I can understand the changes properly.

File: docs/docs/general/02-Getting Started/write-your-first-instruction.md (hunk 1 above)
Comment (JohannesDienst-askui) [reply docs/docs/general/02-Getting Started/write-your-first-instruction.md:5]: Currently we are trying to get our vocabulary sorted out.

The current reasoning is this:
-> Instruction is one singular action that is performed: For example a mouse click
-> Automation: A set of instructions that automate something: For example 2 FA with Github and Android Phone

You are right that we talk of automation on the whole page. We have a set of tickets in JIRA to straighten this out. We just have to start somewhere 😉

File: docs/src/pages/index.js [hunk 4]
```diff
@@ -24,14 +24,7 @@ function HomepageHeader() {
...
 export default function Home() {
-  const { siteConfig } = useDocusaurusContext();
-  return (
-    <Layout>
-      <HomepageHeader />
-      <main>
-        <HomepageFeatures />
-      </main>
-    </Layout>
-  );
```
[the remark this answers is not in the document]
Comment (JohannesDienst-askui) [reply docs/src/pages/index.js:36]: Good catch! I will check it out.

File: docs/docs/general/01-Introduction/why-askui.md (hunk 2 above)
Comment (JohannesDienst-askui) [reply docs/docs/general/01-Introduction/why-askui.md:1]: Yes, with pleasure 😃 

We noticed it takes too long for a user to actually get started. As we are marketing to developers they are usually not interested in the WHY of our product but more in the WHAT CAN IT DO FOR ME question. So by removing the why-askui and the homepage we hope to get them to their first Hello World faster.

File: docs/src/css/custom.scss (hunk 3 above)
Comment (JohannesDienst-askui) [reply docs/src/css/custom.scss]: I checked: The CSS is not easily transferable as the structure is different:

* Theme button -> <button>
* Github Link -> <a>

File: docs/docs/general/02-Getting Started/write-your-first-instruction.md (hunk 1 above)
Comment (adi-wan-askui) [reply docs/docs/general/02-Getting Started/write-your-first-instruction.md:5]: Understood :) 

But to me it still looks like this should stay `Automation` as it is not a single action that is written here...

**Review by adi-wan-askui — APPROVED** [review]
Keep up the good work :)

**Pull request merged**